smart-cards/src/main/java/gov/va/api/health/smartcards/patient/ImmunizationTransformer.java

```diff
@@ -16,18 +21,44 @@ static Reference referenceOnly(@NonNull Reference reference) {
     return Reference.builder().reference(reference.reference()).build();
   }
 
-  Reference location() {
-    if (entry.resource().location() == null) {
-      return null;
+  String occurrenceDateTime() {
+    // only publish the date information. time is not necessary.
+    String odt = entry.resource().occurrenceDateTime();
+    // expect DQ to provide a ISO8601-formatted date
+    if (odt.contains("T")) {
+      odt = Iterables.get(Splitter.on('T').split(odt), 0);
     }
-    return referenceOnly(entry.resource().location());
+    return odt;
   }
 
   Reference patient() {
     // Do not include display
     return referenceOnly(entry.resource().patient());
   }
 
+  List<Immunization.Performer> performer() {
+    if (CollectionUtils.isEmpty(entry.resource().contained())) {
+      return null;
+    }
+    Location location =
+        entry.resource().contained().stream()
+            .filter(r -> r instanceof Location)
+            .map(r -> (Location) r)
+            .findFirst()
+            .orElse(null);
+    if (location == null) {
+      return null;
+    }
+    Reference org = location.managingOrganization();
+    if (org == null || StringUtils.isEmpty(org.display())) {
+      return null;
+    }
```

Comment:
@jblefkove-va in the case that `org.display() == null`, should we follow the `org.reference()` link to get the organization `name` from DQ? or just ignore it?

The Vaccination IG has `performer` as an optional field, with cardinality `0..1`. It's technically not required, but I think we should try our best to include it if we can?

Reply:
I think in production `org.display()` will usually be available; its sufficient for now to use that and not retrieve the whole organization separately